src/services/UserService.ts

```diff
@@ -141,10 +142,45 @@ const getUserPermissions = async (payload: any, token: any): Promise<any> => {
   }
 }
 
+async function getEComStores(token: any): Promise<any> {
+  const baseURL = store.getters['user/getBaseUrl'];
+
+  const params = {
+    "viewSize": 200,
+    "fieldList": ["productStoreId", "storeName"],
+    "entityName": "ProductStoreFacilityDetail",
+    "distinct": "Y",
+    "noConditionFind": "Y",
+    "filterByDate": 'Y',
+  };
+
+  try {
+    const resp = await client({
```

Comment:
We can use api method for fetching productStores and remove passing token explicitly as we have already set token in the state before making this call.